src/style.css

```diff
@@ -1 +1,50 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+main {
+  width: 96px;
```

Comment:
Fixed container sizes can lead to overflow or accidental scroll bars if the content grows larger than the container. Consider using max-width or width in percentages for relative sizing.